dintero-checkout-for-woocommerce.php

```diff
@@ -223,6 +223,28 @@ function () {
 			);
 
 			add_action( 'before_woocommerce_init', array( $this, 'declare_wc_compatibility' ) );
+
+			add_action( 'admin_notices', array( $this, 'wc_zero_decimal_notice' ) );
+		}
+
+
+		/**
+		 * Display a notice if the WooCommerce decimal setting is set to 0.
+		 *
+		 * @return void
+		 */
+		public function wc_zero_decimal_notice() {
+			$decimals_setting = get_option( 'woocommerce_price_num_decimals', 2 );
+
+			if ( 0 == $decimals_setting ) {
+				$article_url = 'https://krokedil.com/dont-display-prices-with-0-decimals-in-woocommerce/';
+				?>
+				<div class="notice notice-warning is-dismissible">
```

Comment:
You have added the class `is-dismissible`, which will handle removing it, but on a reload of the page it will come back. If you want to be able to permanently dismiss this notification you need to store this as a user meta to the WordPress meta, and not display it if its set. If you want i can show you a example of this in some other code we have.